libminifi/test/unit/ProcessorTests.cpp

```diff
@@ -33,10 +35,18 @@ TEST_CASE("Test Find file", "[getfileCreate2]"){
 
 	TestController testController;
 
-	testController.enableDebug();
+	Configure *config = Configure::getConfigure();
 
-	ProvenanceTestRepository repo;
-	TestFlowController controller(repo);
+	config->set(BaseLogger::nifi_log_appender,"rollingappender");
```

Comment:
@apiri  @benqiu2016  Can this be removed? either by Bin or Aldrin on the merge. Logs shouldn't be needed for tests.